index.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+const server = express();
+const logger = require('morgan');
+const helmet = require('helmet');
+const PORT = 4000;
+const dbhelper = require('./dbhelper')
+const bcrypt = require('bcryptjs');
+server.use(express.json());
+server.use(helmet());
+server.use(logger('dev'))
+server.disable("etag");
+
+server.post('/api/register', (req, res) => {
+    const newUser = req.body;
+    console.log(newUser)
+    if (!newUser.password || !newUser.username) {
+        return res
+            .status(404)
```

Comment:
There are more appropriate status codes that could be provided. I would suggest checking this https://www.restapitutorial.com/httpstatuscodes.html out.